.github/workflows/post_summaries.yaml

```diff
@@ -0,0 +1,64 @@
+name: Comment on the pull request
+
+on:
+  workflow_run:
+    workflows: [Health]
+    types:
+      - completed
+
+jobs:
+  upload:
+    runs-on: ubuntu-latest
+    if: >
+      github.event.workflow_run.event == 'pull_request' &&
+      github.event.workflow_run.conclusion == 'success'
+    steps:
+      - name: 'Download artifact'
+        uses: actions/github-script@v3.1.0
```

Comment:
We should always prefer referring to action by their hash and not their semver version (the git tag can be changed out from under us).